### ISSUE TITLE
serving Multi label classification

### ISSUE THREAD

**Issue by slootmaekersdirk** [hanxiao/bert-as-service#342]
Hello,

I have a fine tuned model using sigmoid iso softmax, for multi label classification. My question is should there be any change to your code if I served the finetuned model through HTTP using your service?

thanks
Dirk
